firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -1039,16 +1037,15 @@ class BrowserCoordinator: BaseCoordinator,
     }
 
     func showNativeErrorPage(overlayManager: OverlayModeManager) {
-        // TODO: FXIOS-9641 #21239 Integration with Redux - presenting view
-        let errorpageController = self.errorViewController ?? NativeErrorPageViewController(
+        let errorPageController = NativeErrorPageViewController(
```

Comment:
just an FYI - I found a memory leak when I updated this code for the private homepage, though the reason was an underlying view issue, not necessarily this specific change, so noting here in case the same applies.